client/state/themes/utils.js

```diff
@@ -82,9 +87,15 @@ export function normalizeWporgTheme( theme ) {
 		download_link: 'download',
 	};
 
-	const normalizedTheme = mapKeys( omit( theme, [ 'sections', 'author' ] ), ( value, key ) =>
-		get( attributesMap, key, key )
-	);
+	const normalizedTheme = theme
+		? Object.entries( theme ).reduce( ( acc, [ key, value ] ) => {
+				if ( [ 'sections', 'author' ].includes( key ) ) {
```

Comment:
We previously removed those via `_.omit` which we also refactor away from in this method.